### ISSUE TITLE
Example in docs doesn't compile

### ISSUE THREAD

**Issue by anisbi** [lettuce-io/lettuce-core#1016]
Hey,
I'm following [5.0.1 Connection pooling](https://lettuce.io/core/5.0.1.RELEASE/reference/#_connection_pooling), where in the 'Basic Usage' example you write:

```
RedisClient client = RedisClient.create(RedisURI.create(host, port));
GenericObjectPool<StatefulRedisConnection<String, String>> pool = ConnectionPoolSupport
               .createGenericObjectPool(() -> client.connect(), new GenericObjectPoolConfig());

// executing work
try (StatefulRedisConnection<String, String> connection = pool.borrowObject()) {
    connection.multi();
}
```

This code does not compile, and the error was fixed in [5.1.0 docs](https://github.com/lettuce-io/lettuce-core/wiki/Connection-Pooling-5.1), but it should be:
```
try (StatefulRedisConnection<String, String> connection = pool.borrowObject()) {

    RedisCommands<String, String> commands = connection.sync();
    commands.multi();
```

**Comment by mp911de** [comment]
Thanks for the report. I'm not sure I follow. We fixed the docs for 5.1.0. We do not re-deploy documentation for already released versions because a fair amount of docs lives in the Wiki.

Is that the issue that you meant?

**Comment by anisbi** [comment]
> Thanks for the report. I'm not sure I follow. We fixed the docs for 5.1.0. We do not re-deploy documentation for already released versions because a fair amount of docs lives in the Wiki.
> 
> Is that the issue that you meant?

There is an error in the docs of 5.0.1 which does not appear in 5.1.0, I'm reporting the error of 5.0.1 and noting that it was fixed in 5.1.0.

**Comment by mp911de** [comment]
Thanks for the clarification. The main issue with building and deploying documentation for older versions is that the Wiki isn't tied to the source code version and building older documentation would pull the most recent wiki contents and so it would reflect a different state.

That's why we keep old documentation stable. Prior to lettuce.io, the Wiki was the sole place for documentation.

Closing this ticket as there's nothing we can do here.
